places/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+
+class Place(models.Model):
+    name                         = models.CharField(max_length=100)
+    address                      = models.CharField(max_length=200)
+    phone_number                 = models.CharField(max_length=50, default='')
+    opening_hours                = models.CharField(max_length=200, default='')
+    description                  = models.CharField(max_length=1000, default='')
+    maximum_number_of_subscriber = models.IntegerField(null=True)
+    latitude                     = models.DecimalField(max_digits=16, decimal_places=13)
+    longitude                    = models.DecimalField(max_digits=16, decimal_places=13)
+    able_to_reserve              = models.BooleanField()
+    closed_temporarily            = models.BooleanField()
+    category                     = models.ForeignKey('Category', on_delete=models.CASCADE)
+    region                       = models.ForeignKey('Region', on_delete=models.CASCADE)
+    menu                         = models.ManyToManyField('Menu', through='PlaceMenu', related_name='places')
```

Comment:
```suggestion
    menus                        = models.ManyToManyField('Menu', through='PlaceMenu', related_name='places')
```
`ManyToManyField`를 사용하실 때는 column을 복수(menus)로 만들어 주셔야 합니다.
그래야 place instance에서 menu 목록을 불러올 때 아래와 같이 가독성 있는 코드를 작성할 수 있습니다.

```python
place = Place.objects.get()
..

menus = place.menus.all()
```

ManyToManyField를 사용하는 다른 모든 컬럼에 대해서도 수정해주세요~~!!